### DISCUSSION TITLE
Pinned repo scroll view working weird!

### DISCUSSION THREAD

**Discussion by Hephaest** [community/community#2805] [Mobile] [answered]
[GitHub Mobile][iOS] I install the last version of Github today and view my profile. This issue happened when I was swiping the scroll view from right-most to the left. In other words, when I already scrolled to the last one, and I still swipe from right to left, it will debounce to the second to last. 

**THAT IS WEIRD AND ALSO UNACCEPTABLE.**

Since it happens every time, I assume this is a bug that can be easily fixed (maybe just need to reset the offset). 

Additional Info:
GitHub for iOS, Version 1.3.3 (21290296)
Device: iPhone Xs Max (iOS 14.2)

**Comment by candyho** [comment, marked as answer]
Hey thanks for reporting the bug and sorry for the inconvenience caused. I have created an issue to track this bug in our internal repo. Will keep you posted when this is fixed.
